statsdaemon.go

```diff
@@ -572,6 +576,24 @@ func tcpListener() {
 	}
 }
 
+func heartbeat() {
+	fileName := "heartbeat.monitor"
+	_, err := os.Stat(*heartbeatFilePath + fileName)
+	if os.IsNotExist(err) {
```

Comment:
There are 3 outcomes here, no error (file exists, Chtimes), file not exists error  (create file), other error (i.e. a permission error, folder missing, etc).

Can you add a `log.Fatal` for the 3rd condition?